tests/src/test/java/alluxio/client/rest/S3ClientRestApiTest.java

```diff
@@ -1535,6 +1535,46 @@ public void testTaggingNoLimit() throws Exception {
     testTagging(objectKey, ImmutableMap.copyOf(tagMap));
   }
 
+  @Test
+  public void testCopyObject() throws Exception {
+    final String bucketName = "bucket";
+    final String objectKey = "object";
+    final String targetObject = "/tmp/target";
+
+    String object = CommonUtils.randomAlphaNumString(DATA_SIZE);
+    final String fullObjectKey = bucketName + AlluxioURI.SEPARATOR + objectKey;
+    String copiedObjectKey = bucketName + targetObject;
+    AlluxioURI copiedObjectURI = new AlluxioURI(AlluxioURI.SEPARATOR + copiedObjectKey);
+
+    createBucketRestCall(bucketName);
+    createObject(fullObjectKey, object.getBytes(), null, null);
+
+    // metadata directive = REPLACE, tagging directi
```

Comment:
nit: Typo? Neither directive affect this tests anyway though.

Reply:
fixed